using-mobile-extensions/mobile-core/signal/README.md

```diff
@@ -0,0 +1,174 @@
+
+```java
+public class MobileApp extends Application {
+
+    @Override
+    public void onCreate() {
+        super.onCreate();
+        MobileCore.setApplication(this);
+        MobileCore.configureWithAppId("yourAppId");
+        try {
+            Signal.registerExtension(); //Register Signal extension with Mobile Core
+            MobileCore.start(null);
+        } catch (Exception e) {
+            //Log the exception
+         }
+    }
+}
+```
+
+**Important**: The Signal extension is automatically included in Core by Maven. When you manually install the Signal extension, ensure that you add the `signal-1.x.x.aar` library to your project.
+{% endtab %}
+
+{% tab title="iOS" %}
+In your app's`application:didFinishLaunchingWithOptions`, register the Signal extension with Mobile Core:
+
+#### Objective-C
+
+```objectivec
+- (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions {
+    [ACPCore configureWithAppId:@"yourAppId"];
+    [ACPSignal registerExtension];
+    [ACPCore start:nil];
+    // Override point for customization after application launch.
+    return YES;
+ }
+```
+
+#### Swift
+
+```swift
+func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
+     ACPCore.configure(withAppId: "yourAppId")   
+     ACPSignal.registerExtension()
+     ACPCore.start(nil)
+     // Override point for customization after application launch.
+     return true;
+}
+```
+
+**Important**: The Signal extension is automatically included in the Core pod. When you manually install the Signal extension, ensure that you add the `libACPSignal_iOS.a` library to your project.
+{% endtab %}
+
+{% tab title="React Native" %}
+
+#### JavaScript
+
+```jsx
+import {ACPSignal} from '@adobe/react-native-acpcore';
+
+initSDK() {
+    ACPSignal.registerExtension();
+}
+```
+
+{% endtab %}
+{% endtabs %}
+
+##Define rules in Experience Platform Launch
+
+To define the rules in Experience Platform Launch, complete the steps in [Signal extension and Rules Engine integaration](https://aep-sdks.gitbook.io/docs/using-mobile-extensions/mobile-core/signals/signals-extension-and-rules-engine-integration).
+
+##Implement the Mobile SDK to send PII data to external destinations
+
+To send PII data to external destinations, the `PII` action can trigger the Rules Engine when the configured triggers and traits match. When setting a rule, you can set the `PII` action for a Signal event, so that `collectPii` can trigger the rule and send the `PII` data..
+
+For the information on collectPii and its usage, see collectPii in [AEP Mobile Core API reference Page](https://aep-sdks.gitbook.io/docs/using-mobile-extensions/mobile-core/mobile-core-api-reference#collect-pii)
```

Comment:
Please reword to:
"For more information about `collectPII` and its usage, see [collectPII](https://aep-sdks.gitbook.io/docs/using-mobile-extensions/mobile-core/mobile-core-api-reference#collect-pii).